src/dscim/preprocessing/preprocessing.py

```diff
@@ -44,13 +45,19 @@ def ce_from_chunk(
         raise NotImplementedError("Pass 'cc' or 'no_cc' to reduction.")
 
     if recipe == "adding_up":
-        result = mean_func(
-            np.maximum(
+        if not quantreg:
+            result = mean_func(
+                np.maximum(
+                    calculation,
+                    bottom_code,
+                ),
+                "batch",
+            )
+        else:
+            result = np.maximum(
                 calculation,
                 bottom_code,
-            ),
-            "batch",
-        )
+            )
     elif recipe == "risk_aversion":
```

Comment:
do we need to throw an exception if `recipe == "risk_aversion"` and `quantreg==True` ?